salsah/src/public/js/jquery.resadd.js

```diff
@@ -517,6 +517,7 @@
 												}
 											}
 											tmpele.attr('placeholder', placeholderText);
+											tmpele.attr('autocomplete', 'off');
```

Comment:
sorry, but I do not understand what the attribute `autocomplete` is. Was is already there or did you add it?